sdk/cdn/arm-cdn/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-cdn",
   "author": "Microsoft Corporation",
   "description": "CdnManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "5.2.0",
+  "version": "6.0.0",
```

Comment:
It is not clear to me why the changes in this PR require a major version update. Are there any breaking changes?